From 4f8430abd627c739f2890d31aea1a535208bee00 Mon Sep 17 00:00:00 2001
From: Ismael Luceno <ismael@sourcemage.org>
Date: Sun, 2 Jul 2023 19:38:07 +0200
Subject: [PATCH 3/3] Add exfat and f2fs support

Inspired by patches from Gentoo and Void Linux.

Credit to:
- Bartosz Nitkiewicz <bartosz@nitkiewicz.me>
- Piotr Karbowski <slashbeast@gentoo.org>

Origin: Source Mage
Upstream-Status: Pending
Signed-off-by: Ismael Luceno <ismael@sourcemage.org>
---
 src/fs.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/fs.c b/src/fs.c
index 65ea0c7ab1d3..b4103d0a4455 100644
--- a/src/fs.c
+++ b/src/fs.c
@@ -44,6 +44,8 @@ static struct FS supported_fs[] = {
       1 /* skip detection */
     },
     { "ntfs", "nosuid,nodev,user", 1, "077", ",nls=%s" },
+    { "exfat", "nodev,nosuid,user", 1, "077" },
+    { "f2fs", "nodev,noauto,nosuid,user" },
     { NULL, NULL, 0, NULL, 0}
 };
 
-- 
2.40.1

